sketch/RFUT_data.hpp

```diff
@@ -27,7 +27,7 @@ struct RFUT_data_t {
     RFUT_data_t (int N, skylark::base::context_t& context)
         : _N(N), _creation_context(context) {
 
-        build();
+//        build();
```

Comment:
For what it's worth, tokens such as ``_N``, and, more generally, any symbol beginning with an underscore and followed by a capital letter, are likely to cause compilation problems (and, I would assume, *do* on Cygwin) since they are reserved for the compiler toolchain.

I would recommend switching member variables in classes from the form ``_name`` to the form ``name_`` for this reason.